app/controllers/application_controller.rb

```diff
@@ -1,3 +1,21 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
+
+  helper_method :current_tribunal_case, :current_grounds_for_appeal_document
+
+  def current_tribunal_case
+    @current_tribunal_case ||= TribunalCase.find_by_id(session[:tribunal_case_id])
+  end
+
+  def current_files_collection_ref
```

Comment:
Happy with the `#current_tribunal_case` here like we discussed, but is there a reason why `#current_files_collection_ref` and `#current_grounds_for_appeal_document` need to be in `ApplicationController`? :confused: